atd-etl/populate_cr3_file_metadata/.gitignore

```diff
@@ -1 +1,3 @@
 *pdf
+env
```

Comment:
Thanks for adding this! Makes me think that we should add a `.dockerignore` file in this project folder too just in case someone ever pushes from their own machine.